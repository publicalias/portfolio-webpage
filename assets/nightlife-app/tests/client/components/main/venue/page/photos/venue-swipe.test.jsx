"use strict";

//local imports

const VenueSwipe = require("../../../../../../../scripts/client/components/main/venue/page/photos/venue-swipe");

const { testWrapper } = require("../../../../../test-helpers");

//global imports

const { initTestSnapshot, reactTests } = require("redux/tests/react-tests");

//utilities

const { testShallow } = testWrapper(VenueSwipe);

const testSnapshot = initTestSnapshot(testShallow);

//setup

beforeAll(reactTests.setup);
beforeEach(reactTests.inject(VenueSwipe));

//venue swipe

test("venue swipe should match snapshot", () => testSnapshot());
