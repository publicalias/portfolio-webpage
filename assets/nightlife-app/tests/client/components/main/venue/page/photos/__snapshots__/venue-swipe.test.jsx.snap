// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`venue swipe should match snapshot 1`] = `""`;
